src/transformers/models/nllb/tokenization_nllb_fast.py

```diff
@@ -0,0 +1,298 @@
+# fmt: on
+
+
+class NllbTokenizerFast(PreTrainedTokenizerFast):
+    """
+    Construct a "fast" NLLB tokenizer (backed by HuggingFace's *tokenizers* library). Based on
+    [BPE](https://huggingface.co/docs/tokenizers/python/latest/components.html?highlight=BPE#models).
+
+    This tokenizer inherits from [`PreTrainedTokenizerFast`] which contains most of the main methods. Users should
+    refer to this superclass for more information regarding those methods.
+
+    The tokenization method is `<tokens> <eos> <language code>` for source language documents, and ``<language code>
+    <tokens> <eos>``` for target language documents.
```

Comment:
```suggestion
    The tokenization method is `<tokens> <eos> <language code>` for source language documents, and `<language code>
    <tokens> <eos>` for target language documents.
```

+ missing all the doc of the Args like in the Python tokenizer file.